.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: test
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
This variable is not set (no `strategy` section) so will never be interpolated

Reply:
:facepalm: Copypasta strikes again..